docs/book/src/plugins/available-plugins.md

```diff
@@ -1,5 +1,11 @@
-# Available Plugins
+# Available plugins
 
-This section details how to use the currently available plugins in Kubebuilder. 
+This section describes the plugins supported and shipped in with the Kubebuilder project.
 
-  - [Declarative V1](declarative-v1.md)
```

Comment:
You are removing things from the master branch that you should not
You probably missed it up with the rebase 
You can compare your local branch with master and ensure that you have only what is required to have